src/util/DirectoryExportedClassesLoader.ts

```diff
@@ -6,16 +6,16 @@ import {PlatformTools} from "../platform/PlatformTools";
 export function importClassesFromDirectories(directories: string[], formats = [".js", ".ts"]): Function[] {
 
     function loadFileClasses(exported: any, allLoaded: Function[]) {
-        if (exported instanceof Function) {
+        if (typeof exported === "function") {
```

Comment:
same answer as for array, you can left it however in the codebase everywhere instanceof Function is used and it brings inconsistency with other places